notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<ul><li>The link to the "ranking tutorial" is not working. (I'd remove that sentence if we don't have a ranking tutorial.)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>